content/client-server-api/modules/push.md

```diff
@@ -184,11 +184,13 @@ they are represented as a dictionary with a key equal to their name and
 other keys as their parameters, e.g.
 `{ "set_tweak": "sound", "value": "default" }`.
 
-{{% boxes/note %}}
+###### Historical Actions
+
 Older versions of the Matrix specification included the `dont_notify` and
-`coalesce` actions. These should both be considered no-ops (ignored, not
-rejected) if received from a client.
-{{% /boxes/note %}}
+`coalesce` actions. Homeservers MUST ignore these actions by stripping them
+from any actions array they encounter. This means that setting the actions
+on a rule to e.g. `["dont_notify"]` MUST be equivalent to setting an empty
+actions array.
```

Comment:
Not that I'm keen on making this more complicated but I'm now wondering if this clause should apply to clients as well since they might receive default push rules with `dont_notify` when connecting to an older server? 🤔

Reply:
Yes it probably also applies to clients working with an old server.

Reply:
Ok, I have rephrased the paragraph slightly to include clients as well.